packages/google-tag-manager-analytics/src/index.ts

```diff
@@ -31,6 +31,9 @@ const googleTagManagerAnalytics: GoogleTagManagerAnalytics = {
     },
     googleTagManagerAnalytics: {},
   },
+  libraries: {
+    ...analytics.libraries,
+  },
```

Comment:
Uhm, actually that code ended up there because I had created a `library` in analytics at the beginning of the PR, but then I removed it, so I guess that code is not needed anymore. I'll fix that. :technologist:

Reply:
Done. :heavy_check_mark: